src/plugins/vis_types/pie/public/pie_component.test.tsx

```diff
@@ -120,4 +121,62 @@ describe('PieComponent', function () {
     ]);
     expect(wrapperProps.fireEvent).toHaveBeenCalled();
   });
+
+  it('renders the no results component if all the values are zero', () => {
+    const newVisData = ({
+      type: 'datatable',
+      columns: [
+        {
+          id: 'col-1-1',
+          name: 'Count',
+        },
+        {
+          id: 'col-0-2',
+          name: 'filters',
+        },
+      ],
+      rows: [
+        {
+          'col-0-2': 'Carrier : "JetBeats" ',
+          'col-1-1': 0,
+        },
+        {
+          'col-0-2': 'Carrier : "ES-Air" ',
+          'col-1-1': 0,
+        },
+      ],
+    } as unknown) as Datatable;
+    const newProps = { ...wrapperProps, visData: newVisData };
+    const component = mount(<PieComponent {...newProps} />);
+    expect(findTestSubject(component, 'pieVisualizationError').length).toBe(1);
```

Comment:
```suggestion
    expect(findTestSubject(component, 'pieVisualizationError').text()).toEqual('No results found');
```
It's very nitpicky, feel free to ignore. Just differentiates the added two scenarios :)